test/cmd/node-management.sh

```diff
@@ -109,8 +113,10 @@ __EOF__
   ### kubectl drain update with --dry-run does not mark node unschedulable
   # Pre-condition: node is schedulable
   kube::test::get_object_assert "nodes 127.0.0.1" "{{.spec.unschedulable}}" '<no value>'
-  kubectl drain "127.0.0.1" --dry-run=client
-  kubectl drain "127.0.0.1" --dry-run=server
+  # command - need to use force because pods are unmanaged
+  kubectl drain "127.0.0.1" --dry-run=client --force
+  # TODO: reinstate this when server-side dry run is fixed. See https://github.com/kubernetes/kubernetes/pull/100206
+  # kubectl drain "127.0.0.1" --dry-run=server --force
```

Comment:
As noted in https://github.com/kubernetes/kubernetes/pull/100206#issue-592252171, this command fails with production builds of kubectl. This suite isn't catching it on master because of the aforementioned lack of pods on the target node. Error is:

```
error: error when evicting pods/"$PODNAME" -n "$NAMESPACE": /, Kind= doesn't support dry-run
```